### PR TITLE
Update to use jll artifact

### PR DESCRIPTION
**Pull request by jaakkor2**
* uses jll build by https://github.com/JuliaPackaging/Yggdrasil/tree/master/L/LibSpatialIndex
  * requires Julia v1.3
  * artifact is located at https://github.com/JuliaBinaryWrappers/LibSpatialIndex_jll.jl
* https://github.com/JuliaGeo/LibSpatialIndexBuilder not used anymore

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ os:
   - linux
   - osx
 julia:
-  - 1.0
   - 1
```
Comment (visr) [inline .travis.yml:8]: Could you change the 1.0 to 1.3, such that we always test on the oldest julia version we claim we support, as well as the latest 1.x (1), and nightly.

Same for AppVeyor.

File: Project.toml [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+LibSpatialIndex_jll = "00e98e2a-4326-5239-88cb-15dcbe1c18d0"
 
 [compat]
-BinaryProvider = "0.5"
-julia = "1"
+julia = "1.3"
```
Comment (visr) [inline Project.toml:10]: Here we also need a compat entry for LibSpatialIndex_jll, to make sure that it can be automatically merged in the registry.

**Review by visr — CHANGES_REQUESTED** [review]
This is great, thanks! This is the last build in JuliaGeo to make the switch. Left a few minor comments. Could you also add a line to the readme saying that `libspatialindex` builds are provided, linking to both https://github.com/JuliaPackaging/Yggdrasil/tree/master/L/LibSpatialIndex and https://github.com/JuliaBinaryWrappers/LibSpatialIndex_jll.jl?

@MaximeBouton are you ok with merging this, and archiving https://github.com/JuliaGeo/LibSpatialIndexBuilder?

File: .travis.yml (hunk 1 above)
Comment (jaakkor2) [reply .travis.yml:8]: done

File: Project.toml (hunk 2 above)
Comment (jaakkor2) [reply Project.toml:10]: I set `LibSpatialIndex_jll = "1.8"`

**Review by visr — APPROVED** [review]

**Comment by MaximeBouton** [comment]
This is great, thanks! 
@visr I am fine with archiving the other repo.

**Pull request merged**